src/services/simple-signer-provider.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Wallet, providers, Signer } from 'ethers'
+import { SignerProvider } from './signer-provider.interface'
+
+// TODO Experimental signer
+export class SimpleSignerProvider implements SignerProvider {
```

Comment:
what is this class? I don't see any usage of it (fortunately)

Reply:
It dates back to the beginning when we didn't have the fdp-storage. It's not needed anymore, I'll delete it.